Add syntax highlight extension link to the docs

wspwebben opened the pull request:
Closes #557

Created [extension](https://marketplace.visualstudio.com/items?itemName=webben.browserslist) to support browserslist config files syntax.

Highlights:
1. Comments (starts from #)
2. Version numbers (including `all`, `esr`, and `tp`) + `dead`
3. Market share in percents (50%, 0.5%, .5%)
4. `not, and, or, extends and ,` keywords + `>, >=, <=, <, in` operators
5. [Browser names](https://github.com/browserslist/browserslist#browsers)
6. Headers (`[production]`)
7. External config names (_not present in previews yet_)

I skipped __rare queries__ section for now, because I don't get the syntax rules yet, but I'll look up the browserslist source for inspiration :D

I also added previews into [repository](https://github.com/wspwebben/browserslist-highlight) for both dark and light themes, but for some reason they're not shown nor in vscode, nor in the marketplace.

ai commented:
Highlighting looks great. I put your name to the task.

Things to fix for this PR:

1. Let’s just add a link to `Tools`. Don’t forget to mention that it is for VS COde.
2. Try to find an extension with a screenshot and find how they add images. I think they publish image by GitHub Pages to get absolute URL (like `https://wspwebben.github.io/browserslist-highlight/image.png`) instead of relative URLs (`./image.png`).
3. Do you want to move to Browserslist GitHub org? Rename repo to `browserslist-vscode`, I sent you an invite to move repo.

wspwebben commented:
1. Updated docs, I copied a vscode section from the python extension, but not sure if the the links at the end are necessary.
2. Fixed images, the default branch on GitHub is already `main`, but vsce still uses `master`. You can use relative urls in readme though.
3. I did that, should I now transfer ownership?

ai commented:
> I did that, should I now transfer ownership?

Yeap, you need to use `Transfer ownership` (but it will be still your repository)

ai commented:
@wspwebben don’t forget to respond in issues:
https://github.com/browserslist/browserslist-vscode/issues
